server_addon/maya/client/ayon_maya/api/lib.py

```diff
@@ -4241,3 +4241,78 @@ def get_node_index_under_parent(node: str) -> int:
         return cmds.listRelatives(parent,
                                   children=True,
                                   fullPath=True).index(node)
+
+
+def search_textures(filepath):
+    """Search all texture files on disk.
+
+    This also parses to full sequences for those with dynamic patterns
+    like <UDIM> and %04d in the filename.
+
+    Args:
+        filepath (str): The full path to the file, including any
+            dynamic patterns like <UDIM> or %04d
+
+    Returns:
+        list: The files found on disk
```

Comment:
I wonder if we should update this return type to `list[str | clique.Collection]` since `get_sequence` returns a list of collections - which may be unexpected?
```suggestion
        list[str | clique.Collection]: The files found on disk
```